addons/stock_delivery/views/delivery_view.xml

```diff
@@ -22,6 +22,8 @@
                         <field name="is_return_picking" invisible="1"/>
                         <field name="carrier_id" readonly="state in ('done', 'cancel')" options="{'no_create': True, 'no_open': True}"/>
                         <field name="delivery_type" invisible="True"/>
+                        <field name="tracking_url_show" invisible="True"/>
+                        <field name="carrier_tracking_url" invisible="True"/>
```

Comment:
I would only show it for base_on_rule and fixed delivery rules. For the other, we already have the URLs in the code. Also when they enter a carrier_ref we could directly fetch the attachments on top of it. It would be cool for the user.